open-apparel-registry/open-apparel-registry
# Convert dedupe_indexed_records to a model

## Overview

`dedupe_indexed_records` was previously a standard SQL table, but we're moving it to a django
model, so we can take advantage of django features such as dumpdata and loaddata:
* This PR removes the code that was previously creating the table and index
* the name of the table has changed slightly, from `dedupe_indexed_records` to `api_dedupe_indexed_records`
* the loop that uses SQL to populate the index table is not changing, because there is no easy way to replicate `ON CONFLICT` doing it the django way

Connects #2105

## Testing Instructions

* check out this branch
* run `./scripts/update`. This will run the migration that creates the `api_dedupe_indexed_records` table
* run `./manage/dbshell`
* run `\d api_dedupe_indexed_records` and verify that the table, constraints, and index were created correctly

## Checklist
